### PR TITLE
Fix setting target properties that depend on $<CONFIG>

### PR DESCRIPTION
**Pull request by McMartin**
@MartyLake: please review, thanks!

### CODE REVIEW

File: cmake/Reprojucer.cmake [hunk 1]
```diff
@@ -1554,6 +1554,7 @@ function(__set_common_target_properties target_name)
     endif()
     string(APPEND all_confs_output_name $<$<CONFIG:${configuration_name}>:${output_name}>)
   endforeach()
+  string(APPEND all_confs_output_name $<$<CONFIG:>:${JUCER_PROJECT_NAME}>)
```
Comment (MartyLake) [inline cmake/Reprojucer.cmake]: Should this "advanced" trick inlined in code, instead of the commit's message ?

Comment (McMartin) [inline cmake/Reprojucer.cmake]: You want me to add a comment?

**Comment by MartyLake** [comment]
Maybe a comment would be better suited than a long commit message ? IDK


---
Matthieu TALBOT
neogaldr@gmail.com

+33 629 22 15 70
fr.linkedin.com/in/matthieutalbot
http://www.soundcloud.com/Marty-Lake
http://lifehacker.com/5028808/how-eom-makes-your-email-more-efficient

**Comment by McMartin** [comment]
I took quite some time to write the commit message. Don't you like it? :crying_cat_face: 
More seriously, I'll add a comment before merging. Thanks for the review!

**Comment by MartyLake** [comment]
I love it! I'm more afraid that no one will ever stumble upon it, which
would be a shame! ;-)

**Comment by McMartin** [comment]
@MartyLake: I added a simple comment. If people want to know more, they can look at the git history.

**Comment by MartyLake** [comment]
that's great ! :)

**Pull request merged**